nautobot/dcim/views.py

```diff
@@ -1269,12 +1269,12 @@ def get_extra_context(self, request, instance):
         # v2 TODO(jathan): Replace prefetch_related with select_related
         devices = (
             Device.objects.restrict(request.user, "view")
-            .filter(device_role=instance)
+            .all()
```

Comment:
seems wrong to me, though I presume this entire view is going away soon anyway?

Reply:
`DeviceRoleView` would be removed, so this change is irrelevant, Had to make this change cause it was causing errors